docs/docs/transactions.md

```diff
@@ -97,17 +88,17 @@ You can have concurrent transactions within a sequence of queries or have some o
 
 ### Without CLS enabled
 ```js
-sequelize.transaction(function (t1) {
-  return sequelize.transaction(function (t2) {
+sequelize.transaction(t1 =>
+  sequelize.transaction(t2 =>
     // With CLS enable, queries here will by default use t2
     // Pass in the `transaction` option to define/alter the transaction they belong to.
-    return Promise.all([
-        User.create({ name: 'Bob' }, { transaction: null }),
-        User.create({ name: 'Mallory' }, { transaction: t1 }),
-        User.create({ name: 'John' }) // this would default to t2
-    ]);
-  });
-});
+    Promise.all([
```

Comment:
So what should be the rule? If it doesn't fit in one line, use a return? Should I only do this in the docs or also in the codebase? There are 200+ places where this would have to be changed again